www/app/lib/Zipfile.scala

```diff
@@ -0,0 +1,32 @@
+package lib
+
+import java.io.FileOutputStream
+import java.util.zip.{ZipEntry, ZipOutputStream}
+import com.bryzek.apidoc.generator.v0.models.File
+
+object Zipfile {
+
+  private[this] val UTF8 = "UTF-8"
+
+  def create(dirName: String, files: Seq[File]): java.io.File = {
+    val path = java.io.File.createTempFile(dirName, ".zip")
+    create(path, files, prefix = dirName + "/")
+    path
+  }
+
+  def create(
```

Comment:
Not a big deal here but "they" say it's best to avoid overloading methods in Scala - http://stackoverflow.com/questions/2510108/why-avoid-method-overloading